pkg/plugins/datasource/wrapper/datasource_plugin_wrapper.go

```diff
@@ -79,6 +80,11 @@ func (tw *DatasourcePluginWrapper) Query(ctx context.Context, ds *models.DataSou
 			qr.ErrorString = r.Error
 		}
 
+		if r.MetaJson != "" {
+			metaJson, _ := simplejson.NewJson([]byte(r.MetaJson))
```

Comment:
I think we should log the err if `NewJson` returns an error. Could be very useful for plugin authors to know why it fails.

Reply:
Yes, it's helpful.

Reply:
What do you think, which level is better - Error or Debug?

Reply:
I think Error would be preferred since it something the author should see and fix as soon as possible.